### PR TITLE
style/add prefers reduced motion

### PR DESCRIPTION
**Pull request by lucianmurmurache**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Users that have motion sickness and disable animations on their device, would expect not to see any animations on the platform. We can make use of the [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) feature which is widely supported, when applicable, we can use the [tailwind class](https://tailwindcss.com/docs/transition-property#prefers-reduced-motion) instead.
#### How to enable and test:
- Windows 10: Start > Settings > Ease of Access > Display > Toggle “Show Animations in Windows.”
- macOS: System Preferences > Accessibility > Display> Check “Reduce Motion.”
- iOS: Settings > Accessibility > Motion> Toggle “Reduce Motion.”
- Android: Settings > Accessibility > Toggle “Remove Animations.”

### Related issue

- #722

### CODE REVIEW

**Comment by lucianmurmurache** [comment]
> Looks like we just need to format `tailwind.css` for the frontend test to pass, @lucianmurmurache 😊 Missing the blank last line in the file? Maybe something else, but that's what jumps out :)

For some reason prettier was not auto formatting on my machine, it should be good now. Thank you @andrewtavis

File: frontend/tailwind.config.ts [hunk 1]
```diff
@@ -82,6 +82,9 @@ export default <Partial<Config>>{
           instagram: "rgba(129, 52, 175, 1)",
         },
       },
+      transitionProperty: {
```
Comment (lucianmurmurache) [inline frontend/tailwind.config.ts:85]: Just for a bit of clarity, I added this here as I noticed that `transition-width` was being used but it is not included by default. It now works as expected. 
Example: [PasswordStrength](https://github.com/activist-org/activist/blob/main/frontend/components/PasswordStrengthIndicator.vue#L7)

Comment (andrewtavis) [reply frontend/tailwind.config.ts:85]: Really great to have the attention to these sorts of things, @lucianmurmurache :) Thank you!

File: frontend/components/card/CardConnect.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
         {{ $t("components._global.connect") }}
       </h3>
       <div
-        class="p-1 break-all rounded-lg cursor-pointer transition-all hover:text-light-highlight dark:transition-all dark:hover:text-dark-highlight"
+        class="p-1 break-all rounded-lg cursor-pointer transition-all hover:text-light-highlight motion-reduce:transition-none dark:hover:text-dark-highlight"
```
Comment (andrewtavis) [inline frontend/components/card/CardConnect.vue:8]: Thanks for the fix here and the other place where we're really transitioning on dark mode, @lucianmurmurache! 😄

**Review by andrewtavis — APPROVED** [review]
All looks fantastic on my end, @lucianmurmurache! I'll add a quick note on this to the contribution guidelines and we can close out the new issue! 🚀

**Pull request merged**